README.md

```diff
@@ -1,79 +1,21 @@
 401 JS --  Lab 07 cowsay server
 ===
 
-## Submission Instructions
-  * Work in a fork of this repository
-  * Work in a branch on your fork
-  * Write all of your code in a directory named `lab-` + `<your name>` **e.g.** `lab-duncan`
-  * Submit a pull request to this repository
-  * Submit a link to your pull request on canvas
-  * Submit a question, observation, and how long you spent on canvas  
-  
-## Learning Objectives  
-* Students will be able to implement an HTTP server using Nodejs
+####About the project
```

Comment:
Good README. While instructions aren't explicit, try to get in the habit of establishing how the user should make requests to your API. I.e. what URL should they type into a browser, or what is the command they type in their terminal?